torch/ao/quantization/qconfig_mapping.py

```diff
@@ -73,6 +74,10 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
     else:
         qconfig_transpose = qconfig
 
+    # currently layernorm only supports float weights
+    # we have to add this because otherwise there will be a extra quantize-dequantize pair
+    qconfig_layernorm = QConfig(activation=qconfig.activation, weight=default_placeholder_observer)
```

Comment:
@jcaip for the BC breaking notes, we just need to mention the qconfig for layer_norm, basically we can say something like:

Before:
qconfig_layer_norm = get_default_qconfig("fbgemm")

After:
qconfig_layer_norm = QConfig(activation=default_observer, weight=default_placeholder_observer)